v20.1/migrate-from-mysql.md

```diff
@@ -20,6 +20,31 @@ MySQL strings are case-insensitive by default, but strings in CockroachDB are ca
 
 For more information about the case sensitivity of strings in MySQL, see [Case Sensitivity in String Searches](https://dev.mysql.com/doc/refman/8.0/en/case-sensitivity.html) from the MySQL documentation.  For more information about CockroachDB strings, see [`STRING`](string.html).
 
+### `FIELD` function
+
+The MYSQL `FIELD` function is not supported in CockroachDB. Instead, you can use the [`array_position`](functions-and-operators.html#array-functions) function, which returns the index of the first occurrence of element in the array.
+
+While MYSQL returns 0 when the element is not found, CockroachDB returns `NULL`. So if you are using the `ORDER BY` clause in a statement with the `array_position` function, the caveat is that sort is applied even when the element is not found. As a workaround, you can use the [`COALESCE`](functions-and-operators.html#conditional-and-function-like-operators) operator.
```

Comment:
Can you please move this paragraph above the `COALESCE` example? (I have added another comment with 'X' to mark the spot)

That way the reader can read in a potentially nicer order:

1) intro
2) simple example
3) this paragraph with the more involved description of the workaround
4) the workaround

Reply:
Thank you.